libtcmu_log.c

```diff
@@ -63,6 +60,36 @@ struct log_output {
 	tcmu_log_destination dest;
 };
 
+/* tcmu log dir path */
+static char *tcmu_log_dir = NULL;
+
+/* get the log dir of tcmu-runner */
+char *tcmu_get_log_dir(void)
+{
+	return tcmu_log_dir;
+}
+
+char *tcmu_alloc_and_set_log_dir(const char *log_dir)
+{
+	/*
+	 * Do nothing here and will use the /var/log/
+	 * as the default log dir
+	 */
+	if (!log_dir)
+		return NULL;
+
+	tcmu_log_dir = strdup(log_dir);
+	if (!tcmu_log_dir)
+		tcmu_err("Failed to copy log dir: %s\n", log_dir);
+
+	return tcmu_log_dir;
```

Comment:
Just put this code in tcmu_logdir_create. Also did you mean to name it tcmu_create_log_dir to match the naming format for the other exported function tcmu_free_log_dir?

Reply:
Yes, this has been updated.

With the following two changes:
1, tcmu_free_log_dir --> tcmu_logdir_destroy
2, put tcmu_alloc_and_set_log_dir into tcmu_logdir_create